packages/assert/package.json

```diff
@@ -18,16 +18,16 @@
     "types": "dist/assert.d.ts",
     "files": [
         "dist/**/*.js",
-        "dist/**/*.d.ts"
+        "dist/**/*.d.ts*"
```

Comment:
Why the wildcard at the end here?

Reply:
It is to account for the [declarationmap](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-9.html#new---declarationmap) added in [`a058db4` (#51)](https://github.com/salesforce/sa11y/pull/51/commits/a058db426af0ac1f0e6749434dc438377ed37fa7)